easybuild/easyconfigs/h/Hadoop/Hadoop-2.5.0-cdh5.3.1-native.eb

```diff
@@ -19,12 +19,15 @@ builddependencies = [
     ('Maven', '3.2.3'),
     ('protobuf', '2.5.0'),
     ('CMake', '3.1.3'),
+    ('snappy', '1.1.2', '-intel-2015a'),
```

Comment:
isn't requiring the full `intel/2015a` toolchain a bit much for this?

I suspect `snappy` only requires a compiler to build? So how about limiting this to `GCC/4.9.2` instead for example?

Also, you should correctly specify the deviating toolchain (4th element of the tuple here), rather than abusing the versionsuffix (3rd argument):

```python
    ('snappy', '1.1.2', '', ('GCC', '4.9.2')),
```